lib/src/presentation/widgets/home/app_bar/search_textfeild.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flower_app/core/styles/colors/app_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class SearchTextfeild extends StatelessWidget {
+  const SearchTextfeild({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return TextField(
```

Comment:
There is a custom Text input field try use it

Reply:
in search it is different in design